src/app/models/vehicle-tech-record.model.ts

```diff
@@ -218,6 +218,8 @@ export interface BodyType {
 
 export interface TechRecordModel {
   historicVin?: string;
+  historicPrimaryVrm?: string;
+  historicSecondaryVrms?: string[];
```

Comment:
These values are currently unused, but will likely be used for the ticket regarding archiving VRMs.